etc/licscripts/find_invlic.pl

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/perl
 
 use File::Find;
+use File::Spec;
 use Fcntl ':mode';
 use Getopt::Long;
```

Comment:
Needed further below (see next comment):
```suggestion
use Getopt::Long;
use List::Util qw(none);
```